pages/_error.tsx

```diff
@@ -0,0 +1,46 @@
+// Components
+import { NextPageContext } from 'next';
```

Comment:
```suggestion
import { type NextPageContext } from 'next';
```
This should be a `type` import right?

Reply:
I've seen both, I don't have a preference either way. 
https://nextjs.org/docs/pages/api-reference/functions/get-initial-props

Reply:
> I've seen both, I don't have a preference either way. https://nextjs.org/docs/pages/api-reference/functions/get-initial-props

So the benefit of doing `type`is it allows for better treeshaking (This is mostly relevant when it relates to modules with sideeffects), it bassically tells typescript to extract the type info and not compile the values.